setup.py

```diff
@@ -1,31 +1,44 @@
 from setuptools import setup, find_packages
-import os.path
-
-def read(path):
```

Comment:
Please keep the `read` method, but make it use `with open(...)` to ensure the file is closed properly. This prevents runtime warnings.